app/routes/main.js

```diff
@@ -2,6 +2,6 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
   model() {
-    return Ember.$.getJSON("data/3767MostCommonLyricWordsInDictionary.json");
+    return Ember.$.getJSON("data/752MostCommonLyricWordsInDictionary.json");
```

Comment:
what's this dictionary change? It looks like it's increasing the number of words